conda_content_trust/common.py

```diff
@@ -436,17 +421,18 @@ def checkformat_hex_string(s):
     """
 
     if not isinstance(s, string_types):
-        raise TypeError(
-                'Expected a hex string; given value is not string typed.')
+        raise TypeError('Expected a hex string; given value is not string typed.')
 
     for c in s:
+        # fmt: off
         if c not in [
                 '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', '0',
                 'a', 'b', 'c', 'd', 'e', 'f']:
```

Comment:
By making this change you can also remove `# fmt: off/on`

```suggestion
        if c not in '0123456789abcdef':
```

Reply:
how eminently sensible.  Yes, thanks. 😅